### PR TITLE
BACKLOG-14534 : add JWT Configuration in developerTools panel

### PR DESCRIPTION
**Pull request by Yousria**
## JIRA

https://jira.jahia.org/browse/BACKLOG-14534

### CODE REVIEW

File: src/javascript/jwt/App.jsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ function App(props) {
                     </Typography>
                 </Toolbar>
             </AppBar>
-            <ApolloProvider client={client({contextPath: dxContext.context})}>
+            <ApolloProvider client={client({contextPath: window.contextJsParameters.contextPath})}>
```
Comment (tdraier) [inline src/javascript/jwt/App.jsx:29]: ApolloProvider should not be needed, it's already provided by the app

File: src/main/resources/tools/developer/jwtConfiguration.jsp [hunk 2]
```diff
@@ -22,7 +22,7 @@
 
 <html>
 <head>
-    <script src="${pageContext.request.contextPath}/modules/security-filter-tools/javascript/apps/security-filter-jwt.js" ></script>
+    <script src="${pageContext.request.contextPath}/modules/security-filter-tools/javascript/apps/jahia.bundle.js" ></script>
```
Comment (tdraier) [inline src/main/resources/tools/developer/jwtConfiguration.jsp:25]: Does it work ? The jahia.bundle.js is built with a DLL and cannot be called by itself.  We don't need this JSP anyway, so you can remove it

File: package.json [hunk 3]
```diff
@@ -2,20 +2,30 @@
   "name": "@jahia/security-filter",
   "version": "2.0.0",
   "scripts": {
-    "dev": "./node_modules/.bin/webpack --watch --inline",
-    "build": "./node_modules/.bin/webpack --config=webpack.build.config.js"
+    "dev": "yarn webpack --watch",
+    "build": "yarn webpack"
```
Comment (tdraier) [inline package.json:6]: Can't we have simply 
```suggestion
    "dev": "webpack --watch",
    "build": "webpack"
```
?

File: package.json [hunk 4]
```diff
@@ -2,20 +2,30 @@
...
-    "@jahia/react-apollo": "^1.0.0-beta.20180605080720",
-    "@jahia/react-material": "^1.0.0-beta.20180710133148",
-    "@jahia/redux": "^1.0.0-beta.20180628145924",
+    "@jahia/apollo-dx": "^1.1.0",
+    "@jahia/design-system-kit": "^1.1.8",
+    "@jahia/i18next": "^1.1.0",
+    "@jahia/moonstone": "^0.16.1",
+    "@jahia/react-apollo": "^1.1.0",
+    "@jahia/react-material": "^3.0.0",
+    "@jahia/redux": "^1.1.0",
```
Comment (tdraier) [inline package.json:27]: redux does not seem to be used

File: src/javascript/jwt/App.jsx [hunk 5]
```diff
@@ -26,7 +26,7 @@ function App(props) {
...
                 <DxContext.Provider value={dxContext}>
                     <TokenManager dxContext={dxContext}/>
                 </DxContext.Provider>
```
Comment (tdraier) [inline src/javascript/jwt/App.jsx:32]: DxContext.Provider is actually not used neither
```suggestion
                <TokenManager dxContext={dxContext}/>
```

File: src/main/resources/tools/developer/jwtConfiguration.jsp (hunk 2 above)
Comment (Yousria) [reply src/main/resources/tools/developer/jwtConfiguration.jsp:25]: yes the JSP isn't needed anymore, it's not used

File: src/javascript/jwt/App.jsx (hunk 5 above)
Comment (Yousria) [reply src/javascript/jwt/App.jsx:32]: was not sure, but yes I like to remove unused lines :)

**Review by tdraier — APPROVED** [review]

File: src/javascript/jwt/App.jsx (hunk 5 above)
Comment (tdraier) [reply src/javascript/jwt/App.jsx]: you can also remove the DxContxt file :-)

**Pull request merged**